docs/current/contribute.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Contribute
+permalink: /contribute/
+nav_order: 6
+---
+## Contribute
+
+### Report Problems
+If you have what looks like a bug, please use the [Github issue tracking system](https://github.com/ossf/scorecard/issues). Before you file an issue, please search existing issues to see if your issue is already covered.
```

Comment:
This github tracking link was wrong when 1.1 released, use the working docs link here instead.
```suggestion
If you have what looks like a bug, please use the [Github issue tracking system](https://github.com/google/osv-scanner/issues). Before you file an issue, please search existing issues to see if your issue is already covered.
```